docs/book/features/vsphere_csi_migration.md

```diff
@@ -0,0 +1,175 @@
+| Tolerate Datacenter name, Datastore name and Storage policy name changes | Not Supported | supported |
+| Volume Encryption | Not supported | Supported |
+| Kubernetes Cluster spread across multiple vCenter Servers | Supported | Not supported |
+| Kubernetes Cluster spread across multiple datacenters within a vCenter Server | Supported | Supported |
+| Volume Topology(with `waitForFirstConsumer`) | Supported | Supported |
+| Thick disk provisioning | Supported on all datastore types (vSAN, VVOL, VMFS and NFS) | Supported only on vSAN Datastore using Storage Policy Capability - `Object space reservation` |
+| Raw Block Volume | Supported | Not supported |
+| Inline volumes in Pod spec | Supported | Not supported |
+
+In addition to the above feature comparison, one of the most important things customers need to consider is that Kubernetes will deprecate In-tree vSphere volume plugin and it will be removed in the future Kubernetes releases.
+Volumes provisioned using vSphere in-tree plugin do not get the additional new features supported by the vSphere CSI driver.
+
+Kubernetes has provided a seamless procedure to help migrate in-tree vSphere volumes to a vSphere CSI driver. After in-tree vSphere volumes migrated to vSphere CSI driver, All subsequent operations on migrated volumes are performed by the vSphere CSI driver.
+Migrated vSphere volume will not get additional capabilities vSphere CSI driver supports.
+
+## Consider followings before turning on Migration
+
+- vSphere CSI Migration is released with `beta` feature-gate in Kubernetes 1.19. Refer [release note announcement](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#csi-migration---azuredisk-and-vsphere-beta).
+- Kubernetes 1.19 release has deprecated vSAN raw policy parameters for the in-tree vSphere Volume plugin and these parameters will be removed in a future release. Refer [deprecation announcement](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#deprecation)
+- Following vSphere in-tree StorageClass parameters will not be supported after enabling migration.
+  - `hostfailurestotolerate`
+  - `forceprovisioning`
+  - `cachereservation`
+  - `diskstripes`
+  - `objectspacereservation`
+  - `iopslimit`
+  - `diskformat`
+- Storage Policy should not be renamed or deleted after enabling migration. Volume migration requires storage policy to be present to successfully register volume as Container Volume in vSphere.
+- Datastore should not be renamed after enabling migration. Volume migration relies on the datastore name retrieved from the volume path available in the Persistent Volume Source.
+- For statically created vSphere in-tree Persistent Volume Claims and Persistent Volumes, make sure to add following annotations. Statically provisioned in-tree vSphere volumes can't be migrated to CSI without following these annotations.
```

Comment:
add following annotations before migration.

Reply:
made it clear

> For statically created vSphere in-tree Persistent Volume Claims and Persistent Volumes, make sure to add the following annotations before enabling migration